src/fireEvent.ts

```diff
@@ -53,7 +53,7 @@ const findEventHandler = (
   eventName: string,
   callsite?: any,
   nearestTouchResponder?: ReactTestInstance
-) => {
+): Function | null => {
```

Comment:
Maybe we should define a type for event handlers?. We could start with non-generic version like `type EventHandler = (...args: any) => unknown`.